### PR TITLE
Update docs regarding the docker run suggestion issue

### PR DESCRIPTION
**Pull request by vickyhella**
Issue link: https://github.com/harvester/docs/issues/123

### CODE REVIEW

**Review by guangbochen — APPROVED** [review]
LGTM

File: docs/rancher/virtualization-management.md [hunk 1]
```diff
@@ -8,14 +8,33 @@ keywords:
...
 
-1. As a prerequisite, you will need to create a Rancher server with `v2.6.3` or above for Harvester `v1.0.0` integration. For testing purposes, you can spin up a Rancher server using the following `docker run` command:
-    ```
-    $ sudo docker run -d --restart=unless-stopped -p 80:80 -p 443:443 --privileged rancher/rancher:v2.6.3
-    ```
-
-    !!!note
-        For a production Rancher server setup, please refer to the official Rancher [docs](https://rancher.com/docs/rancher/v2.6/en/quick-start-guide/deployment/).
-
+1. As a prerequisite, you will need to create a Rancher server with `v2.6.3` or above for Harvester `v1.0.0` integration. In production, use one of the following guides to deploy and provision Rancher and a Kubernetes cluster in the provider of your choice:
```
Comment (nunix) [inline docs/rancher/virtualization-management.md:11]:
```suggestion
1. As a prerequisite, Harvester `v1.0.0` integration requires Rancher server `v2.6.3` or above. In production, use one of the following guides to deploy and provision Rancher and a Kubernetes cluster with the provider of your choice:
```

**Review by nunix — COMMENTED** [review]
Few modifications to the providers' introductions.

**Comment by guangbochen** [comment]
Hi @btat @jtravee can u guys  PTAL, thanks!

**Review by nunix — APPROVED** [review]

**Pull request merged**